src/nativeComponents/WheelPicker.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {requireNativeComponent, Picker} from 'react-native';
+import { Constants} from '../helpers';
+
+const WheelPicker = requireNativeComponent('WheelPicker', null);
+
+
+class WheelPickerView extends React.Component {
+  constructor(props) {
+    super(props);
+    this.onValueChange = this.onValueChange.bind(this);  //eslint-disable-line
+  }
+
+  state = {
+    ...this.stateFromProps(this.props),
+  }
+
+  UNSAFE_componentWillReceiveProps(nextProps) {  //eslint-disable-line
```

Comment:
What's that?

Reply:
Since wheel picker is basically PickerIOS but for android, I thought it would make sense to add this addition as a pull request to react-native. To make this easier I am using almost exactly the same logic that react-native uses in PickerIOS. Specifically, I think that this is named "UNSAFE_componentWillReceiveProp" because this lifecycle will probably be deprecated in the future, but I am not sure.